tests/goth/assets/docker/docker-compose.yml

```
@@ -0,0 +1,38 @@
+version: "3.3"
+
+services:
+
+    router:
+        image: yagna-goth
+        entrypoint: /usr/bin/ya-sb-router
+        command: ["-l", "tcp://0.0.0.0:7477"]
+
+    proxy-nginx:
+        # A service that runs `nginx` and routes API calls to
+        # the `mitmproxy` instance running as part of the test
+        # harness on the host machine
+        image: proxy-nginx
+        ports:
+            - "6000:6000"
+
+    ethereum:
+        image: docker.pkg.github.com/golemfactory/gnt2/gnt2-docker-yagna:483c6f241edd
+        ports:
+            - "8545:8545"
+
+    zksync:
+        image: docker.pkg.github.com/golemfactory/yagna-zksync/yagna-zksync-mock:7a25ed913d4a
```

```suggestion
        image: docker.pkg.github.com/golemfactory/yagna-zksync/yagna-zksync-mock:f6d0cf02f6bc
```

I just realised updating these image hashes is going to be troublesome once tests are split out into project repos. :x